### DISCUSSION TITLE
catagories not showing

### DISCUSSION THREAD

**Discussion by surendraalamuru22** [learndevopsonline/learndevopsonline#1827] [Q&A]
<img width="960" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/159264263/5bf9d668-2329-4446-aeed-488d2710f52b">

<img width="960" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/159264263/d10f272d-3b4a-4385-a5f9-a28e7abcf69c">

**Comment by chaithanya1812** [comment]
Hello surender,
you have not loaded the schema into mongo database, can you check once..
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/b4834d4a-d271-4b57-8747-0b4c4ffe3bc3)

**Reply by surendraalamuru22** [reply]
i was loaded schema manually also

**Reply by chaithanya1812** [reply]
can show that how you was doing that loading schema

**Reply by surendraalamuru22** [reply]
<img width="960" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/159264263/90209ad1-ded7-4280-bdcd-88f05cf0c0fb">

**Reply by surendraalamuru22** [reply]
![Uploading image.png…]()

**Reply by surendraalamuru22** [reply]
<img width="960" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/159264263/66690c5e-da43-4426-b920-0dd3f76f8bc1">

**Comment by chaithanya1812** [comment]
change all ports 80-->8080, It will works..
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/3d72f1d6-9131-4db6-b750-83e49db309b9)

**Reply by surendraalamuru22** [reply]
ok chaitanya

**Comment by surendraalamuru22** [comment]
ya chaithanya its working fine thank you
